### PR TITLE
[13.0][UPD] noupdate_changes.xml files up to day

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Updating.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/migrations/13.0.1.1/noupdate_changes.xml [hunk 1]
```diff
@@ -35,6 +35,9 @@
   <record id="account_reconcile_model_template_comp_rule" model="ir.rule">
     <field name="domain_force">['|',('company_id','=',False),('company_id', 'in', company_ids)]</field>
   </record>
+  <record id="action_open_settings" model="ir.actions.act_window">
```
Comment (pedrobaeza) [inline addons/account/migrations/13.0.1.1/noupdate_changes.xml:38]: Is this really a noupdate relevant thing?

Comment (MiquelRForgeFlow) [reply addons/account/migrations/13.0.1.1/noupdate_changes.xml:38]: Relevant or not, it appears. I can add commented if you prefer.

Comment (pedrobaeza) [reply addons/account/migrations/13.0.1.1/noupdate_changes.xml:38]: Well, I'm just surprised that's noupdate. Let's keep it.

**Review by pedrobaeza — APPROVED** [review]
Thanks for watching on these things that I don't have into account

**Pull request merged**